renderer/components/Home/WalletLauncher.js

```diff
@@ -7,12 +7,16 @@ import { withRouter } from 'react-router-dom'
 import { Box, Flex } from 'rebass/styled-components'
 import styled from 'styled-components'
 import parse from 'lndconnect/parse'
+import { isEqual } from 'lodash'
```

Comment:
This should use `import isEqual from 'lodash/isEqual` syntax in order to prevent bundle bloat.